SDWebImage/SDWebImageDownloaderConfig.m

```diff
@@ -37,9 +37,9 @@ - (id)copyWithZone:(NSZone *)zone {
     config.sessionConfiguration = [self.sessionConfiguration copyWithZone:zone];
     config.operationClass = self.operationClass;
     config.executionOrder = self.executionOrder;
-    config.urlCredential = [self.urlCredential copyWithZone:zone];
-    config.username = [self.username copyWithZone:zone];
-    config.password = [self.password copyWithZone:zone];
+    config.urlCredential = self.urlCredential;
+    config.username = self.username;
+    config.password = self.password;
```

Comment:
@dreampiggy Should I need to revert all these strings back?

Reply:
Just OK to change. So I merge it now.